src/NumericalAlgorithms/DiscontinuousGalerkin/MetricIdentityJacobian.cpp

```diff
@@ -69,6 +74,15 @@ void metric_identity_jacobian_impl(
     const tnsr::I<DataVector, 3, Frame::Inertial>& inertial_coords,
     const Jacobian<DataVector, 3, Frame::Logical, Frame::Inertial>&
         jacobian) noexcept {
+  // The 3d case is handled separately because this actually requires a buffer.
+  // Basically, in 2d you can get into the situation where you have 2 unused
+  // components (and thus 2 buffers) and so taking the eta derivatives you have
+  // plenty of space for transposing. In 3d all components of the Jacobian
+  // depend on either eta or zeta derivatives and so you can't use one of them
+  // as a buffer to do a transpose.
+  //
+  // The 3d implementation is what should be generalized to higher dimensions if
+  // the need arises.
```

Comment:
This was squashed into the wrong commit (should go in commit that adds impl)

Reply:
:( hopefully done correctly now:P